tests/integration/defs/accuracy/test_step3p7_gsm8k_canary.py

```
@@ -0,0 +1,477 @@
+    is the GSM8K question wrapped with ``"Question: ... Answer:"`` so the
+    model emits a chain-of-thought followed by ``#### <answer>``.  The
+    gold answer is extracted from the ``answer`` field as the integer
+    following the GSM8K ``#### N`` marker.
+    """
+    try:
+        from datasets import load_dataset
+    except ImportError:
+        return _FALLBACK_GSM8K_PROBES[:num_examples]
+    try:
+        ds = load_dataset("gsm8k", "main", split="test")
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)